homework-1.rb

```diff
@@ -0,0 +1,10 @@
+require "csv"
+
+array = []
+File.open("homework-1_numbers.csv") do |file|
+	file.each do |line|
+		array += line
```

Comment:
Close! It should be

```rb
array << line
```

Arrays work a little differently, `<<` means pushing or adding an item into the array.